clients/cli/tests/test.rs

```diff
@@ -442,3 +436,34 @@ async fn update_metadata(raise_minimum_delegation: bool) {
         .unwrap();
     assert!(status.success());
 }
+
+#[tokio::test]
+#[serial]
+async fn display() {
```

Comment:
This can be done in follow-up work, but is there a plan to create a test for the create on-ramp command?